Makefile

```diff
@@ -1,5 +1,5 @@
 SRC_JS_FILES := $(shell find src/components src/js -type f -name '*.js')
-SRC_JS_FILES_FOR_COMPILER = $(shell sed -e ':a' -e 'N' -e '$$!ba' -e 's/\n/ --js /g' .build-artefacts/js-files | sed 's/^.*base\.js //')
+SRC_JS_FILES_FOR_COMPILER = $(shell sed -e ':a' -e 'N' -e '$$!ba' -e 's/\n/ --js /g' .build-artefacts/js-files | sed -r 's/(.*)/ --js \1/g')
```

Comment:
I cannot remember if I had to do this for the update of closure compiler or to solve goog.require issue for print (#2404). I just tested it seems to work fine without. Maybe this is a particularity of our environment.

PR updated.